### PR TITLE
For #277: add tests to PrimaryOutputTests

### PR DESCRIPTION
**Pull request by paulodamaso**
For #277: add tests to PrimaryOutputTests
- added tests for to check `file()`, and `tag()`  methods of `PrimaryOutput` and add method to check generated primary.xml for libdeflt test   rpm package

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#296](https://github.com/artipie/rpm-adapter/issues/296) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/156e0e57-78c0-4b45-8880-7e8fdfa6a130); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/rpm/pkg/PrimaryOutputTest.java [hunk 1]
```diff
@@ -72,4 +72,58 @@ void createsPrimaryForAbc(@TempDir final Path temp) throws IOException {
...
+            Files.readAllBytes(res),
+            CompareMatcher.isIdenticalTo(
+                Files.readAllBytes(
+                    Paths.get("src/test/resources-binary/repodata/libdeflt-primary.xml")
+                )
+            ).ignoreWhitespace()
+            .ignoreElementContentWhitespace()
+            .normalizeWhitespace()
+            .withNodeFilter(
+                node -> !"file".equals(node.getLocalName())
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/rpm/pkg/PrimaryOutputTest.java:121]: @paulodamaso are you sure we need to filter `file` nodes? As I can see, example xml does not have `file` list, xml we generate has?

File: src/test/java/com/artipie/rpm/pkg/PrimaryOutputTest.java [hunk 2]
```diff
@@ -29,16 +29,16 @@
...
 import org.xmlunit.matchers.CompareMatcher;
 
 /**
  * Test for {@link PrimaryOutput}.
  * @since 0.10
- * @todo #241:30min Extend this test: add test methods to check `file()`, `maid()` and `tag()`
- *  methods of `PrimaryOutput` and add method to check generated primary.xml for libdeflt test
- *  rpm package.
+ * @todo #277:30min Extend this test: add test methods to check `maid()` method of
+ *  PrimaryOutput`
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/rpm/pkg/PrimaryOutputTest.java:41]: @paulodamaso test for `maid()` method should only check that `maid()` returns correct instance of `XmlMaid`, maybe you could add here please? I do not think that it should be done in separate PR...

**Comment by Vatavuk** [comment]
@paulodamaso there are conflicts

**Comment by 0crat** [comment]
@vatavuk[/z](https://www.0crat.com/u/vatavuk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by olenagerasimova** [comment]
@paulodamaso ping

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable
